.gitignore

```diff
@@ -27,3 +27,7 @@ docs/src
 _site
 .sass-cache
 .jekyll-metadata
+
+# Junk
+.DS_Store
+
```

Comment:
I would just add this to your global gitignore. It doesn't need to live here, because it's a platform level concern.